packages/mini-apps-ui-kit-react/src/components/PhoneField/PhoneField.tsx

```diff
@@ -113,7 +113,10 @@ export const PhoneField = forwardRef<HTMLDivElement, PhoneFieldProps>(
         value,
         countries: extendedCountries,
```

Comment:
On your proposal to pass the list of available countries via props I would say we can accept a `countries` prop that is forwarded to the react-phone-number-input component